plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/DockerInstanceRuntimeInfo.java

```diff
@@ -82,17 +83,20 @@
     protected static final String SERVER_CONF_LABEL_PATH_SUFFIX     = ":path";
 
     private final ContainerInfo               info;
-    private final String                      containerHost;
+    private final String                      containerExternalHostname;
+    private final String                      containerInternalHostname;
     private final Map<String, ServerConfImpl> serversConf;
 
     @Inject
     public DockerInstanceRuntimeInfo(@Assisted ContainerInfo containerInfo,
-                                     @Assisted String containerHost,
+                                     @Assisted String containerExternalHostname,
```

Comment:
Not named assisted injection doesn't work when several arguments have the same class